Client/Coordinators/EnhancedTrackingProtectionCoordinator.swift

```diff
@@ -43,7 +43,15 @@ class EnhancedTrackingProtectionCoordinator: BaseCoordinator,
     }
 
     func start() {
-        router.push(enhancedTrackingProtectionMenuVC)
+        enhancedTrackingProtectionMenuVC.modalPresentationStyle = .custom
```

Comment:
In the legacy code we had"
```
        if UIDevice.current.userInterfaceIdiom == .phone {
            etpVC.modalPresentationStyle = .custom
```
So I think in the case of iPad we still want to use the popover instead of the custom one.